strings/en_US/messages.po

```diff
@@ -4037,3 +4037,18 @@ msgstr "This Dojo had events recently. Join the Dojo to get notified when ticket
 
 msgid "Please select at least one ticket"
 msgstr "Please select at least one ticket"
+
+msgid "Email should be in the format: janedoe@example.com"
+msgstr "Email should be in the format: janedoe@example.com"
+
+msgid "Password is required"
+msgstr "Password is required"
+
+msgid "There was a problem logging in:"
```

Comment:
@DanielBrierton doesn't that needs to integrate the variable to avoid ordering issues?

Reply:
Well, actually all that displays is `There was a problem logging in: Invalid email or password!`, so perhaps we should only have the one overall string anyway.